ui/admin/app/routes/scopes/scope/credential-stores/credential-store/credential-libraries/index.js

```diff
@@ -1,3 +1,13 @@
 import Route from '@ember/routing/route';
 
-export default class ScopesScopeCredentialStoresCredentialStoreCredentialLibrariesIndexRoute extends Route {}
+export default class ScopesScopeCredentialStoresCredentialStoreCredentialLibrariesIndexRoute extends Route {
+  // =methods
+
+  setupController(controller) {
+    const scope = this.modelFor(
```

Comment:
Again it's a credential store, not a scope, so I'd update the naming here and in the template.

Reply:
updated1